osu.Game/Screens/Play/Player.cs

```diff
@@ -508,7 +508,7 @@ private void onBreakTimeChanged(ValueChangedEvent<bool> isBreakTime)
 
         private void updateGameplayState()
         {
-            bool inGameplay = !DrawableRuleset.HasReplayLoaded.Value;
+            bool inGameplay = !DrawableRuleset.HasReplayLoaded.Value && !GameplayState.HasPassed;
             bool inBreak = breakTracker.IsBreakTime.Value || DrawableRuleset.IsPaused.Value || GameplayState.HasFailed;
```

Comment:
Hmm with failing being considered as "break" I'm not sure the `ScoreManager.PauseImports` change is entirely correct. I'd say _any_ sort of gameplay completion should transition the state to `NotPlaying`...?

Reply:
The reason I didn't originally add failing is that I thought we'd want things like imports to stay blocked while at the fail screen (else they may continue into the start of the next play). I think it's also quite crippling when imports run during `HighPerformanceSession`. But maybe this is an edge case we just allow to happen for now?

Reply:
> we'd want things like imports to stay blocked while at the fail screen

Except for the part where you can request the failed play to be imported at the fail screen which will presumably deadlock now.

Reply:
Yeah, probably need a test and just doing as you say for now.